### ISSUE TITLE
why the playbackrate is >1?

### ISSUE THREAD

**Issue by publicocean0** [mbebenita/Broadway#153]
when i see your demos have 2 problems
the videos run too much fast .

just first time i loaded page i saw glishes in video

**Comment by soliton4** [comment]
hi @publicocean0 and thank you for checking out our project.
would you be willing in participating to fix the issues you pointed out?

**Comment by publicocean0** [comment]
yes ... but before i need to understand the real possibilities to use in production this library.

I tested your demo using notebook and it is toomuch fast,
using my ipad it too much slow. 

The worring problem is above all for ipad. It is slow for algorithm problem or for a hardware problem?
Pratically did you test in mobile device if your solution can work?

**Comment by soliton4** [comment]
right now broadway is just a decoder. it can be used for low latency video streams with no fixed fps.
i would love to see it evolve towards a real player (no pun intended)
first step would be a fixed framerate implementation. some timer + requestAnimationFrame would be my recommendation.
as for dependencies. broadway will run on every ecmascript compatible device that also supports canvases.
webgl can be used to accellerate painting but is not mandatory

**Comment by publicocean0** [comment]
My idea was to make a step forward and to use a javascript decoder+canvas
for implementing a MSE specifications.
In this way we can skip the IOS safari and IE missing and all browsers can
use MSE. Pratically a mse shim using just javascript, NO FLASH:)
Making it .... adobe can tell bye bye  to his resistence to innovation
finally.
The problem that i saw is the code for now is designed for passing the
entire file, not chunks. This solution can not works in a real world....
video file are heavy and must be loaded in streaming.

2017-06-03 15:46 GMT+02:00 soliton4 <notifications@github.com>:

**Comment by soliton4** [comment]
@publicocean0 i would like broadway to function without forcing the user to use encrypted streams

**Comment by publicocean0** [comment]
MSE is not for encryption , is just for loading a video file using chunks
